tests/CommunityToolkit.Mvvm.UnitTests/Test_AsyncRelayCommand.cs

```diff
@@ -225,4 +226,30 @@ public async Task Test_AsyncRelayCommand_WithConcurrencyControl()
 
         Assert.IsFalse(command.IsRunning);
     }
+
+    [TestMethod]
+    public async Task Test_AsyncRelayCommand_ThrowingTaskBubblesToUnobservedTaskException()
```

Comment:
Should an async call calling another async call which fails behave the same way? Do we want to have a test case for a more complex setup like that?

Reply:
It should work the same way, yeah, the method would just throw when awaiting it, so the task would still end up being falted regardless, just with a different stack trace and exception (but we just care about some general failure) 🙂